lib/sober_swag/types/comma_array.rb

```diff
@@ -0,0 +1,17 @@
+module SoberSwag
+  class Types
+    ##
+    # An array that will be parsed from comma-separated values in a string, if given a string.
+    module CommaArray
+      def self.of(other)
+        SoberSwag::Types::Array.of(other).constructor { |val|
```

Comment:
This should be a do/end

Reply:
We call `.meta` on it, and I hate `end.thing`.